semantic/tasks/admin/distributions/update.js

```diff
@@ -0,0 +1,179 @@
+/*******************************
+          Update Repos
+*******************************/
+
+/*
+
+ This task update all SUI individual distribution repos with new versions of distributions
+
+  * Commits changes from create repo
+  * Pushes changes to GitHub
+  * Tag new releases if version changed in main repo
+
+*/
+
+var
+  gulp           = require('gulp'),
+
+  // node dependencies
+  console        = require('better-console'),
+  fs             = require('fs'),
+  path           = require('path'),
+  git            = require('gulp-git'),
+  githubAPI      = require('github'),
+  requireDotFile = require('require-dot-file'),
+
+  // admin files
+  github         = require('../../config/admin/github.js'),
+  release        = require('../../config/admin/release'),
+  project        = require('../../config/project/release'),
+
+
+  // oAuth configuration for GitHub
+  oAuth          = fs.existsSync(__dirname + '/../../config/admin/oauth.js')
+    ? require('../../config/admin/oauth')
+    : false,
+
+  // shorthand
+  version = project.version
+;
+
+module.exports = function(callback) {
+
+  var
+    index = -1,
+    total = release.distributions.length,
+    timer,
+    stream,
+    stepRepo
+  ;
+
+  if(!oAuth) {
+    console.error('Must add oauth token for GitHub in tasks/config/admin/oauth.js');
+    return;
+  }
+
+  // Do Git commands synchronously per distribution, to avoid issues
+  stepRepo = function() {
+
+    index = index + 1;
+    if(index >= total) {
+      callback();
+      return;
+    }
+
+    var
+      distribution         = release.distributions[index],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 6.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182945)